app/Livewire/TaskModal.php

```
@@ -22,22 +23,28 @@ class TaskModal extends Component
     }
 
     #[On('open-task-modal')]
-    public function open(?int $taskId = null): void
+    public function open(?int $taskId = null, ?string $prefillDeadline = null): void
     {
         $this->resetErrorBag();
         $this->form->reset();
+        $this->reset(['task', 'editTaskId', 'createTask']);
 
         if ($taskId) {
             $this->task = Task::findOrFail($taskId);
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\Task::findOrFail().


(staticMethod.notFound)